.github/workflows/rspec_tests.yaml

```diff
@@ -0,0 +1,41 @@
+---
+name: RSpec test
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  rspec_tests:
+    name: ${{ matrix.cfg.os }}(ruby ${{ matrix.cfg.ruby }})
+    strategy:
+      matrix:
+        cfg:
+          - {os: ubuntu-18.04, ruby: '2.5'}
+          - {os: ubuntu-18.04, ruby: '2.7'}
+          - {os: ubuntu-18.04, ruby: '3.0'}
+          - {os: ubuntu-18.04, ruby: 'jruby-9.2.17.0'}
+          - {os: windows-2019, ruby: '2.5'}
+          - {os: windows-2019, ruby: '2.7'}
+          - {os: windows-2019, ruby: '3.0'}
+          - {os: windows-2019, ruby: 'jruby-9.2.17.0'}
```

Comment:
We can drop jruby on windows, because we just want to make sure when this code will work when running in puppetserver and we don't support puppetserver on windows.